### PR TITLE
update api

### PR DESCRIPTION
**Pull request by festinalente91**
I updated get items api
- changed to callback chain style
- changed return type to json
- chagned test case

### CODE REVIEW

**Comment by ovekyc** [comment]
LGTM

**Comment by fguy** [comment]
Do not merge master into remote branch. Use rebase if you need.

File: src/test/node/items/controllers.js [hunk 1]
```diff
@@ -242,4 +254,4 @@ test('delete all item from database', t => {
       });
     });
   });
-});
+});
```
Comment (fguy) [inline src/test/node/items/controllers.js]: New line here

**Comment by fguy** [comment]
LGTM besides of nit

**Comment by fguy** [comment]
LGTM

**Pull request merged**